faust/stores/rocksdb.py

```diff
@@ -26,6 +26,7 @@
     cast,
 )
 
+import pkg_resources
```

Comment:
On second thought, I really don't need this import. I should probably remove it before merging.